UIforETW/Utility.cpp

```diff
@@ -547,9 +770,9 @@ std::wstring GetBuildTimeFromAddress(void* codeAddress)
 	// Print out the module information. The %.24s is necessary to trim
 	// the new line character off of the date string returned by asctime().
 	// _wasctime_s requires a 26-character buffer.
-	wchar_t ascTimeBuf[26];
+	wchar_t ascTimeBuf[26] = {0};

```

Comment:
That's expecting a lot. The CRT does the null termination and it would be pretty ballsy of the compiler to assume that of the CRT. It would have to read and interpret the /analyze annotations, but it actually ignores them during regular compiles. And that's good, because they are supposed to be analysis hints, not optimization hints.